android/app/src/main/java/org/fossasia/openevent/core/discount/DiscountViewHolder.java

```diff
@@ -0,0 +1,82 @@
+package org.fossasia.openevent.core.discount;
+
+import android.support.v7.widget.RecyclerView;
+import android.text.TextUtils;
+import android.view.View;
+import android.widget.TextView;
+
+import org.fossasia.openevent.R;
+import org.fossasia.openevent.common.utils.Utils;
+import org.fossasia.openevent.data.DiscountCode;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+public class DiscountViewHolder extends RecyclerView.ViewHolder {
+
+    @BindView(R.id.discount_code)
+    protected TextView discountCodeTextView;
+
+    @BindView(R.id.discount_code_value)
+    protected TextView discountCodeValue;
+
+    @BindView(R.id.discount_is_active)
+    protected TextView discountIsActive
+            ;
+    @BindView(R.id.discount_max_quantity)
+    protected TextView discountMaxQuantity;
+
+    @BindView(R.id.discount_min_quantity)
+    protected TextView discountMinQuantity;
+
+    @BindView(R.id.discount_tickets)
+    protected TextView discountTicket;
+
+    @BindView(R.id.discount_url)
+    protected TextView discountUrl;
+
+    @BindView(R.id.discount_used_for)
+    protected TextView discountUsedFor;
+
+
+    public DiscountViewHolder(View itemView) {
+        super(itemView);
+        ButterKnife.bind(this, itemView);
+
+
+    }
+
+    public void bindDiscountCode(DiscountCode discountCode) {
+
+        String code = Utils.checkStringEmpty(discountCode.getCode());
+        String discountUrl = Utils.checkStringEmpty(discountCode.getDiscount_url());
+        String tickets = Utils.checkStringEmpty(discountCode.getTickets());
+        String usedFor = Utils.checkStringEmpty(discountCode.getUsed_for());
+
+        int minQuantity = discountCode.getMin_quantity();
+        int maxQuantity = discountCode.getMax_quantity();
+        float value = discountCode.getValue();
+
+        setStringFieldWithPrefix(discountCodeTextView, code, "");
+        setStringFieldWithPrefix(discountCodeValue, "Value : "+value, "");
+        setStringFieldWithPrefix(discountMinQuantity, "Minimum Quantity : "+minQuantity, "");
+        setStringFieldWithPrefix(discountMaxQuantity, "Maximum Quantity :"+maxQuantity, "");
```

Comment:
Spacing between +

I'd recommend using template strings

Reply:
do you mean use of strings.xml
like this :
<string name="MinQuanString">Minimum Quantity : %1$d </string>

Reply:
Yes